### DISCUSSION TITLE
Climate Change Projections of ETCCDI Variables

### DISCUSSION THREAD

**Discussion by LawalKA** [WCRP-CORDEX/discuss#6] [Q&A] [answered]
Hi, is it possible to generate datasets for the projections of ETCCDI variables form the downscaled CMIP6?

**Comment by jesusff** [comment]
Dear Lawal, I took the freedom to move your question to this general discussion, where I think it is more appropriate.

Regarding your question, just to clarify, do you mean publishing these derived indices on ESGF, as the rest of basic CORDEX output variables?

**Comment by LawalKA** [comment]
Yes, as this will make our search for these kinds of datasets
seamless. The gesture will be highly appreciated.

**Comment by larsbuntemeyer** [comment, marked as answer]
@LawalKA that's a good point, as far as i know, there are no climate indicator datasets published in ESGF so far, probably simply due to a missing understanding or agreeement on meta data conventions. There is some work going on regarding this issue, e.g., here: 

* [clix-meta](https://github.com/clix-meta/clix-meta)

and an implementation here:

* [xclim CF indicators](https://xclim.readthedocs.io/en/stable/indicators_api.html#cf-standard-indices)

There will defenitely be some indicators computed from CORDEX-CMIP6 datasets and i am not sure, if there could be a coordinate community effort simply due to different access and availibilty of datasets.
